compiler/compile/OMRCompilation.cpp

```diff
@@ -2713,3 +2714,17 @@ OMR::Compilation::CompilationPhaseScope::~CompilationPhaseScope()
    {
    _comp->restoreCompilationPhase(_savedPhase);
    }
+
+TR::Region &OMR::Compilation::aliasRegion()
+   {
+   return self()->_aliasRegion;
+   }
+
+void OMR::Compilation::invalidateAliasRegion()
+   {
+   if (self()->_aliasRegion.bytesAllocated() > (0.75) * TR::Region::initialSize())
```

Comment:
How was the `0.75` threshold chosen?  Can this constant be pulled out into at least a `#define` so its not so hidden?